python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -441,19 +441,19 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
             log.warning("doMaskBrightObjects is set to True, but brightObjectMask not loaded")
         self.processResults(retStruct.coaddExposure, inputData["brightObjectMask"], outputDataId)
 
-        if self.config.doWriteArtifactMasks and supplementaryData:  # branch to see if running CompareWarp.
-            dataIds = [ref.dataId for ref in inputs.warpRefList]
-            psfMatchedDataIds = [ref.dataId for ref in supplementaryData.warpRefList]
-
-            if dataIds != psfMatchedDataIds:
-                supplementaryData.warpRefList = reorderAndPadList(
-                    supplementaryData.warpRefList, psfMatchedDataIds, dataIds
-                )
```

Comment:
I have no idea what I was thinking here. These are not used in this block, and `supplementaryData.warpRefList` is already reordered and padded by the time it gets here.

Reply:
They were `None`s yes. But the `supplementaryData` was never used in this block or after, making these lines unnecessary.